.github/workflows/go.yml

```
@@ -12,32 +12,23 @@ jobs:
         run: |
           go get -v -t -d ./...
 
-      - name: Format
-        run: diff -u <(echo -n) <(gofmt -d .)
-
       - name: Build
         run: go build -v .
 
-      - name: Test and Coverage
+      - name: Test
         run: |
```

actionlint

shellcheck reported issue in this script: SC2155:warning:1:8: Declare and assign separately to avoid masking return values

(shellcheck)